mscore/pm.cpp

```diff
@@ -18,6 +18,8 @@
 //  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 //=============================================================================
 
+#include <QtCore/qglobal.h>
```

Comment:
The #if defined(Q_OS_WIN) was not properly being evaluated without this line.  There is probably a better way to get that define...I just need to figure out.